opds_import.py

```diff
@@ -275,7 +276,15 @@ class OPDSImporter(object):
 
     COULD_NOT_CREATE_LICENSE_POOL = (
         "No existing license pool for this identifier and no way of creating one.")
-   
+
+    NAME = ExternalIntegration.OPDS_IMPORT
+    FIELDS = [
+        {
+            "key": Collection.EXTERNAL_ACCOUNT_ID_KEY,
+            "label": _("URL"),
+        }
+    ]
+
```

Comment:
The URL of an OPDS feed is analogous to the "library ID" of an Overdrive account, and not to the URL of the Overdrive web service root, so I understand why we did that, and on balance I'd like to keep it on the Collection rather than in the integration.